tests/spirv/pointer-2.slang

```diff
@@ -0,0 +1,70 @@
+//TEST:SIMPLE(filecheck=CHECK_SPV): -entry vertexMain -stage vertex -emit-spirv-directly -target spirv
+//TEST:SIMPLE(filecheck=CHECK_GLSL): -entry vertexMain -stage vertex -target glsl
```

Comment:
Can you add a test that -target spirv -emit-spirv-via-glsl and check for “;Glslang” to verify the glsl is valid and can be compiled by glslang.